### PR TITLE
initial commit - added code to template and if single get or set in c…

### PR DESCRIPTION
**Pull request by ajaymathur**
…lass

Work in progress PR

| Q                        | A 
| ------------------------ | ---
| Fixed Issues             |  Fixes https://github.com/babel/babel/issues/6258 
| Patch: Bug Fix?          | No
| Major: Breaking Change?  | No
| Minor: New Feature?      | Yes
| Tests Added/Pass?        |  No 😞 
| Spec Compliancy?         | 
| License                  | MIT
| Doc PR                   | 
| Any Dependency Changes?  | No

### CODE REVIEW

**Comment by ajaymathur** [comment]
Still working on this.

Please drop initial reviews and guide me if I doing something in the wrong way. :)

File: packages/babel-plugin-transform-es2015-classes/src/vanilla.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ const noMethodVisitor = {
   },
 };
 
+const transformToObjectDefineProperty = template(`
+  Object.defineProperty(constructor, prop, {
```
Comment (hzoo) [inline packages/babel-plugin-transform-es2015-classes/src/vanilla.js]: So if you want to replace something in a template you need it to be all caps

Comment (ajaymathur) [reply packages/babel-plugin-transform-es2015-classes/src/vanilla.js]: Thanks will make this change and will keep this in mind. :)

**Comment by ajaymathur** [comment]
Closing this PR in favour of https://github.com/babel/babel/pull/7038

**Pull request closed without merging**